### ISSUE TITLE
Option to show no background at X responsive breakpoint

### ISSUE THREAD

**Issue by Florrpan** [beaverbuilder/feature-requests#380]
**You usually have a large backgrund for desktop, and at lower breakpoints you don't need a background at all**

At smaller screens, you more than often can't see an entire background image selected for desktop, and maybe you don't have a reason to show that background on a phone because you show the background image as an image module at these breakpoints instead.

I usually end up needing to add a blank PNG or something to hide the large background on smaller breakpoints. And figured, why isn't there an option to just "disable" the background image at these breakpoints altogether?

**Describe the solution you'd like**
Add a function to allow NULL image for selected breakpoints. Right now it just passes the large breakpoint downwards.
This would also allow for quick speed optimization of the website instead of workarounds with blank images.

**Comment by AJDesignsCO** [comment]
Yes! ... and when at a breakpoint it would be great to see  if it is inheriting its background from a breakpoint above. When tabbing through many row and column layouts responsively it would be helpful to know if the current element has a background set from a breakpoint above it. As it is now, there is no indication at smaller screen sizes that it has a background image at all, unless we briefly look at the large screen size. 

![bg](https://github.com/user-attachments/assets/867d5aba-a89b-431e-83a4-89e89970a9aa)

Being able to see that a background is set at the breakpoint and be able to set it to NULL would be helpful.
